docs/getting-started.md

```diff
@@ -91,38 +91,39 @@ Explore our CLI to find more queries you can run.
 ### Submitting a transaction
 
 
-1. Generate an account
+1. Generate a key-pair
 
-    You need an account to author transactions.
-    An account is a randomly generated key pair identified by a name of your choice.
-    To generate an account, run:
+    You need a key-pair to author transactions.
+    The key-pairs are randomly generated and identified by a name of your choice.
+    To generate a new key-pair, run:
 
     ```bash
-    radicle-registry-cli account generate <account_name>
+    radicle-registry-cli key-pair generate <name>
     ```
 
-    This command will give you the SS58 address for your new account.
-    E.g. generating an account named 'neo':
+    This command will give you the SS58 address for your new key-pair.
+    E.g. generating a key-pair named 'neo':
 
     ```bash
-    radicle-registry-cli account generate neo
-    ✓ Account generated successfully
-    ℹ SS58 address: 5HWP48i9TuP2VrRZrNeb6QzYpdUSE9BvyaptEqqHrLfH8ZPd
+    radicle-registry-cli key-pair generate neo
+    ✓ Key-pair generated successfully
+    ⓘ SS58 address: 5HWP48i9TuP2VrRZrNeb6QzYpdUSE9BvyaptEqqHrLfH8ZPd
     ```
 
-    You can always obtain the addresses of all your accounts by running:
+    You can always obtain the addresses of all your key-pairs by running:
 
     ``` bash
-    radicle-registry-cli account list
+    radicle-registry-cli key-pair list
     ```
 
 2. Request RADs
 
-    To interact with the ledger, you first need to get some RAD into your account to have enough
-    funds to pay for transactions fees.
+    To submit transactions, you first need to get some RAD into your key-pair
+    account to have enough funds to pay for transactions fees.
```

Comment:
This introduces the unexplained term “key pair account”. I think just “account” should suffice.